mobile-first/css/index.css

```diff
@@ -434,71 +415,83 @@ h5 {
 
 .hm-container1 {
     display: flex;
-    justify-items: space-between;
-    margin: 40px 0;
+    flex-direction: column;
+    /* justify-items: space-between; */
+    /* margin: 40px 0; */
 }
 
-@media (max-width: 500px) {
-    .hm-container1 {
 
-        flex-direction: column;
-    }
+.hm-container1 {
+
+    flex-direction: column;
+}
+
+.features {
+    /* padding-left: 20px; */
+    margin-top: 10px;
 }
 
 .about {
-    padding-left: 20px;
+    margin: 15px 0 10px 0;
+
 }
 
-@media (max-width: 500px) {
-    .about {
-        padding-left: 0;
+@media (min-width: 500px) {
```

Comment:
Go ahead and play with the styles. I check under the hood on a bunch of sites I go to just to see how they are doing things